aws/table_aws_resource_explorer_search.go

```diff
@@ -0,0 +1,231 @@
+package aws
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/aws/arn"
+	"github.com/aws/aws-sdk-go-v2/service/resourceexplorer2"
+	"github.com/aws/aws-sdk-go-v2/service/resourceexplorer2/types"
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAWSResourceExplorerSearch(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_resource_explorer_search",
+		Description: "AWS Resource Explorer Search",
+		List: &plugin.ListConfig{
+			Hydrate: awsResourceExplorerSearch,
+			IgnoreConfig: &plugin.IgnoreConfig{
+				// UnauthorizedException error thrown for below cases in Resource Explorer
+				// 1. Default view is not present in the region queried
+				// 2. Credentials doesn't have access to the view used for searching
+				// 3. Cross-account or cross-region view is used for searching
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"UnauthorizedException"}),
```

Comment:
Can you please remove? I'd rather the table use be a bit noisy in the beginning as AWS throws this error for mostly all cases, and then we can add improved error handling/messages on a case by case basis.